KTLOSAQ40Loot.json

```diff
@@ -374,7 +374,7 @@
 			"wowID":"22399"
 		},
 		"Qiraji Bindings of Command":{
-			"prio":["non-DTP/R13 DPS Warriors->Rogues->Warriors","EP/GP"],
+			"prio":["SilkyJohnsn(2)x2","Shadowsfade(1)x2","Killstep(2)x2","Chart(1)x2","Stopbegging(2)x2","BenDriller(1)x2","non-DTP/R13 DPS Warriors->Rogues->Warriors","EP/GP"],
```

Comment:
I thought we weren't assigning prio for non-super bis stuff in R2

Reply:
i think its fair the rogues get theirs, since the tanks got everything and all the dps warriors got theirs

Reply:
also i know silky has shoulders, same with killstep

Reply:
rogues have generally been full loot council in both raids, and then started being just full loot council for progress raiders when that became a thing. Because of the way our tier sets worked, it made the most sense for previous tiers, for this one it doesn't matter as much but I do think that it is one of the reasons we have our highest performing rogues willing to split pretty evenly between raids.

Reply:
I think I'm ok with this if just change it to a single token rather than x2.
1) We said we would be light on LC non-super bis stuff, and this might be little heavy handed for R2 to digest.
2) The tanks / OTs still want the shoulders, so this spans beyond rogues (it's the best plate dps shoulders in the game, and arguably a minor upgrade for tanks).

Reply:
I'd rather just get full sets on rogues ASAP. The shoulders specifically people might run alone and the evisc damage bonus is helpful in pve and a big reason to use the set in pvp. 

Loot council started with the Sunday rogues in phase 1 before it spread to the rest of the guild just because it made the most sense for the guild. I'm not going to stop LCing stuff because of anything related to Dasmi or Moxie, but if there are people who are affected by it and unhappy I'm willing to listen, although I'd prefer to put them into the LC rotation over putting the pieces to EP/GP.

My understanding is we've already passed the shoulder tokens to warriors without drake talon pauldrons. I'd prefer to get people into the LC rotation to ensure the tier gets stacked on rogues when the pieces are going to rogues, but I think that partially depends on what you mean by arguably minor upgrade.

If it's definitely an upgrade, maybe a good one, maybe a small one, I think it's worth working in people who want it. If it is actually a strict upgrade for tanks, they can take it with tank prio.
If it's arguably an upgrade, maybe a small upgrade, maybe a side-grade then I think people can wait.

I've talked with Snydicate, and my understanding is that for him, specifically, the reason he wants the last few pieces of conq are just because he like having full sets. Which is great, but I think the items should be prioritized to people who will actually wear them for the majority of fights in a raid. Everyone will get all the tier items eventually and right now it's jut a matter of who gets them first.

I also expect that warrior and rogue t3 will be LC'd in both raids due to wanting to get the 4 pc set bonuses on the right people and then as many people as possible ASAP.

Reply:
1. 5/5 DD is a tiny dps upgrade for PvE, and I think you're the only rogue on that list that actively PvPs so rushing 5/5 for anyone else is not a strong argument.

2. For R2, the shoulders have been EPGP and have actually gone to rogues via EPGP.  We've received quite a few questions around "I thought those were DPS warrior prio" and our response has been "In R2, these are just EPGP".  Now if we're changing that, that's fine, but then we definitely need to slot in the DPS warriors for parity with R1.

3. For tanks, it's a minor upgrade (-4 agi, 1 dodge, +4 stam, +30 armor, +4 def, +1 hit) vs DTP and I would not invoke tank prio for a small upgrade like this.  However, the extra hit does allow the flexibility of dropping hit else where (eg. personally I would gain 18 stam 14 def by dropping accuria once I get shoulders).  They're also the best DPS/Off tank shoulders in the game, and unless you are Snydicate/Frau, you will be DPSing as a warrior.

Side note:  Snydicate is a bit of an idiot when it comes to... everything.  I wouldn't count too much on his insight on why a piece is good/bad for tanks :P